test/testdata

```diff
@@ -133,7 +133,8 @@ Scan      Jaguar       JaguarScanTest       basicJaguar8.3        dvb_scan.out
 
 SP        ADF         ADFSPTest             basicADF2007.01       dvb_sp_b.adfout
 SP        ADF         ADFSPTest             basicADF2013.01       dvb_sp_b.adfout
-SP        DALTON      DALTONSPTest          basicDALTON-2013      b3lyp_energy_dvb_sp.out
+SP        DALTON      DALTONSPTest          basicDALTON-2013      sp_b3lyp_dvb.out
+SP        DALTON      DALTONSPTest          basicDALTON-2013      sp_rhf_dvb.out
```

Comment:
Well, it's like this. DALTON takes two files as input: a .dal files that contains the calculation instructions and a .mol files that described the molecule (both are in the repo, and they are repeated in the output). The default output filename is the concatenation of the basenames of the two (in the order I gave aboce). So, it seemed natural to do it this way.

Since we are at the initial stages of the parser, it might make sense to regularize this to the cclib convention. I am actually for this, but did not care enough to decide myself. If you also think it's a good idea, I will rename.